### PR TITLE
Fix SKU reviews layout issues  in small space

### PR DESCRIPTION
**Pull request by alistat**

### CODE REVIEW

**Comment by alistat** [comment]
There is a typo in the file partner_sku_reviews.coffee : `usefull`
I suggest we remove the term altogether and replace it with `helpful` (reviews) in accordance with the API response.

File: src/plugins/partner_sku_reviews.coffee [hunk 1]
```diff
@@ -1088,16 +1092,25 @@ class PartnerSkuReviews
       padding: 15px;
     }
 
+    ##{flavor}-product-reviews-extended .sa-review-header {
+      display: flex;
+      flex-direction: row;
```
Comment (kosdiamantis) [inline src/plugins/partner_sku_reviews.coffee]: Add the `-ms` prefixed version of Flexbox:

```css
display: -ms-flexbox;
-ms-flex-direction: row;
```

File: src/plugins/partner_sku_reviews.coffee [hunk 2]
```diff
@@ -1139,6 +1154,10 @@ class PartnerSkuReviews
       -webkit-text-fill-color: currentColor;
     }
 
+    ##{flavor}-product-reviews-extended .sa-review-date-wrap {
```
Comment (kosdiamantis) [inline src/plugins/partner_sku_reviews.coffee]: I suggest to use `-container` instead of `-wrap`

Comment (kosdiamantis) [reply src/plugins/partner_sku_reviews.coffee]: Ignore me we use `-wrap` all over the place 😄

File: src/plugins/partner_sku_reviews.coffee [hunk 3]
```diff
@@ -1205,16 +1224,19 @@ class PartnerSkuReviews
     ##{flavor}-product-reviews-extended .sa-review-sentiment {
       display: block;
       position: relative;
-      padding: 0 0 1px;
+      padding: 3px 0 3px 30px;
```
Comment (kosdiamantis) [inline src/plugins/partner_sku_reviews.coffee]: I suggest: `padding-left: 25px`

File: src/plugins/partner_sku_reviews.coffee [hunk 4]
```diff
@@ -1205,16 +1224,19 @@ class PartnerSkuReviews
...
       content: "";
       display: inline-block;
+      position: absolute;
+      top: 2px;
+      left: 0;
       width: 21px;
       height: 21px;
-      margin-right: 4.2px;
-      vertical-align: middle;
+      font-size: 21px;
```
Comment (kosdiamantis) [inline src/plugins/partner_sku_reviews.coffee]: ✂️

File: src/plugins/partner_sku_reviews.coffee [hunk 5]
```diff
@@ -1205,16 +1224,19 @@ class PartnerSkuReviews
...
+      margin-right: 0.2em;
```
Comment (kosdiamantis) [inline src/plugins/partner_sku_reviews.coffee]: ✂️ 

💭 `position: absolute`

**Pull request merged**